meteor/client/ui/Shelf/Keyboard/KeyboardPreview.tsx

```diff
@@ -204,7 +204,7 @@ export const KeyboardPreviewKey: React.FC<IKeyboardPreviewKeyProps> = React.memo
 				}
 			)}
 			style={{
-				fontSize: props.keyPosition.width >= 0 ? (props.keyPosition.width || 1) + 'em' : undefined,
+				width: props.keyPosition.width >= 0 ? (props.keyPosition.width || 1) + 'em' : undefined,
```

Comment:
It is suppose to use `em`, and not ´rem´?

Reply:
The keyboard preview is subject to the dashboard panel's `scale` property, which is applied as font-size on the `KeyboardPreviewPanel`, therefore everything inside of it should remain scaled accordingly